docs/latest/design/roadmap.md

```diff
@@ -32,13 +32,18 @@ contributing to the project.
 - Achieve Gateway API conformance (e.g. routing, LB, Header transformation, etc.)- [Issue #65][65].
 - Setup a CI/CD pipeline- [Issue #63][63].
 
-### [v0.3.0][v0.3.0]: Drive Advanced Features through Extension Mechanisms
+### [v0.3.0][v0.3.0]: Driving Advanced Features through Extension Mechanisms
 
-- Global Rate Limiting
-- AuthN/AuthZ- [Issue #336][336].
-- Lets Encrypt Integration
+- Support extended Gateway API fields.
+- Support experimental Gateway APIs such as TCPRoute [Issue #643][643], UDPRoute [Issue #641][641] and GRPCRoute [Issue #642][642].
+- Rate Limiting [Issue #670][670].
+- Authentication [Issue #336][336].
```

Comment:
IMHO this is overpromising. I would like to understand which maintainers will support these features. We should not depend on the community to deliver roadmap features. I'm in the process of updating #336, but this only covers the AuthN design. From recent feedback (#675 #677) and past experience, the AuthN design alone will take time. Since AuthN is laying the foundation for our approach to Gateway API extensions, this may cause a delay in starting #670. What was the conclusion for delaying RL for RLQS?

Reply:
@danehans I assigned #670 to myself, I can design many aspects of it while waiting on the extension decision.
reg RLQS
* we dont have any info on when it will release
* will it be a better fit than Envoy's native RLS for the Gateway case ?
* will it be more stable  ?
so will start the initial design with Envoy RLS, and can raise a follow up issue to consider moving to RLQS when we
have enough data to answer above questions